Makefile

```diff
@@ -95,9 +95,17 @@ help-extended: #HELP Display extended help.
 #SECTION Development
 
 .PHONY: lint
-lint: $(GOLANGCI_LINT) #HELP Run golangci linter.
+lint: lint-custom $(GOLANGCI_LINT) #HELP Run golangci linter.
 	$(GOLANGCI_LINT) run --build-tags $(GO_BUILD_TAGS) $(GOLANGCI_LINT_ARGS)
 
+.PHONY: custom-linter-build
+custom-linter-build:
+	cd ./hack/ci/custom-linters/cmd/ && go build -o ../../../../bin/custom-linter
+
+.PHONY: lint-custom
+lint-custom: custom-linter-build
+	CGO_ENABLED=1 go vet -vettool=./bin/custom-linter ./...
```

Comment:
```suggestion
.PHONY: lint-custom
lint-custom: custom-linter-build
	go vet -tags=$(GO_BUILD_TAGS) -vettool=./bin/custom-linter ./...
```

Reply:
I notice we also have a `make vet` target. Which seems duplicate since we run `golangci-lint` which includes `govet`.

Just need to enable it here: https://github.com/operator-framework/operator-controller/blob/dcf50b86b5a48608d72b487990dfa88416caf26e/.golangci.yaml#L27

Reply:
We can remove the make vet and enable to golangci
But I understand that `go vet -tags=$(GO_BUILD_TAGS) -vettool=./bin/custom-linter ./...` only calls the new linter not the the default code of go vet 
I will do this cleanup of make vet and golangci in another PR for sure